### PR TITLE
Pract.02: Add 01 02 06 | Teodora Tsaneva

### PR DESCRIPTION
**Pull request by tedy11**
Teodora Tsaneva
IS 4 5MI0700155

### CODE REVIEW

File: Task01.cpp [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        cout << "Error";
+        return -1;
+    }
+    int sum = a + b + c;
+    int product = a * b * c;
+    writeFile << sum << endl << product;
+    writeFile.close();
+    ifstream readFile("result.txt");
+    print(readFile);
+    readFile.close();
```
Comment (khatoni) [inline Task01.cpp]: You should have function one for reading and one for writing, both with is open check

File: file_Task02.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+sdjfhj
```
Comment (khatoni) [inline file_Task02.txt]: Do not upload .txt files!

File: file_Task06.txt [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    }
+    char bufer[1024];
+    while (true)
+    {
+    if (myFile.eof())
+        break;
+    myFile.getline(bufer, 1024);
+   
+    printFile << bufer;
+    printFile << endl;
```
Comment (khatoni) [inline file_Task06.txt]: You do not need another stream

File: file_Task06.txt [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    {
+        cout << "Error";
+        return -1;
+    }
+    char bufer[1024];
+    while (true)
+    {
+    if (myFile.eof())
+        break;
+    myFile.getline(bufer, 1024);
```
Comment (khatoni) [inline file_Task06.txt]: 1024 should be const variable

File: file_Task06.txt [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    if (!myFile.is_open())
+    {
+        cout << "Error";
+        return -1;
+    }
+    char bufer[1024];
+    while (true)
+    {
+    if (myFile.eof())
+        break;
```
Comment (khatoni) [inline file_Task06.txt]: this should be the condition in the while loop

File: Task06.cpp [hunk 6]
```diff
@@ -0,0 +1,34 @@
+﻿#include <iostream>
```
Comment (khatoni) [inline Task06.cpp]: Sorry, the comment are in the .txt file :D

File: Task02.cpp [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+    int counter = 0;
+    while (!myFile.eof())
+    {
+        char buff[100000];
+        myFile.getline(buff, 100000);
+        int i = 0;
+        while (buff[i] != '\0')
+        {
+            counter++;
+            i++;
```
Comment (khatoni) [inline Task02.cpp]: Use tellg, seekg

**Review by khatoni — CHANGES_REQUESTED** [review]

**Comment by Justsvetoslavov** [comment]
Check uploaded solutions.
You can re-open the PR or open a new one if you have better solution or questions.

**Pull request closed without merging**